ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -80,8 +80,8 @@ gcp_connection::put_object(const std::string &object_key, const std::string &fil
 
     // Check if file has been successfully uploaded.
     if (!metadata) {
-        std::cerr << "Upload failed: " << metadata.status() << std::endl;
-        return -1;
+        std::cerr << "Upload failed: " << metadata.status().message() << std::endl;
```

Comment:
Can we add the `object_key` to this message?

Reply:
Updated: https://github.com/wiredtiger/wiredtiger/commit/203c6b83eee8f9eead661087edf53e4493e4b1eb